content/blog/2020-01-themis-0-1-0/index.Rmarkdown

```diff
@@ -0,0 +1,105 @@
+---
+title: themis 0.1.0
+author: Emil Hvitfeldt
+date: '2020-01-13'
+slug: themis-0-1-0
+description: 
+  themis 0.1.0 is now available on CRAN. Provides additional steps for `recipes`
+  to deal with unbalanced data.
+categories:
+  - package
+tags:
+  - tidymodels
+photo: 
+  url: https://unsplash.com/photos/RtDwtRDvYQg
+  author: Roman Kraft
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're chuffed to announce the release of [themis](https://github.com/tidymodels/themis) is now on CRAN. [themis](https://tidymodels.github.io/themis/) implements a collection of new steps for the [recipes](https://github.com/tidymodels/recipes) package to deal with unbalanced data. themis is still in early development so any and all feedback is highly appreciated.
+
+```{r, message=FALSE}
+library(modeldata)
+library(recipes)
+library(themis)
+```
+
+In a classification context, a dataset is said to be unbalanced if there is an unequal number of observations in each class. Many models perform best when the number of observations is equal and will thus have a tendency to struggle with unbalanced data.
+
+The steps in this package can be divided into two camps:
+
+- Ones that remove observations from the majority class(es) and
+- Ones that add observations to the minority class(es).
+
+One is not restricted to do only one action and is thus able to mix and match by for example first removing observations from the majority class followed by adding observations to the minority class to achieve the balance you want.
```

Comment:
By "One" here you're referring to the reader or the one doing the modelling, right? English is sorely lacking in the indefinite-pronoun department (meaning "one"), and I think it's trickier here because you've just used "Ones" in the list above to refer to types of steps. Thus, I would fully switch to using "You" here (e.g. _"You can do more than one action, and, thus, are able to mix and match by, for example, removing observations..."_)

Reply:
I was indeed trying to use an indefinite pronoun, will switch to "you"